src/helpers.ts

```diff
@@ -0,0 +1,34 @@
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
+
+export const prepareTodos = (
+  todos: Todo[],
+  status: string,
+  query: string,
+): Todo[] => {
+  let visibleTodos = [...todos];
+
+  switch (status) {
+    case TodoStatus.ACTIVE:
+      visibleTodos = visibleTodos.filter(todo => !todo.completed);
+      break;
+    case TodoStatus.COMPLETED:
+      visibleTodos = visibleTodos.filter(todo => todo.completed);
+      break;
+    case TodoStatus.ALL:
+    default:
+      break;
+  }
+
+  if (query) {
+    const normalizedQuery = query
+      .split(' ')
+      .map(word => word.trim().toLowerCase())
+      .join(' ');
```

Comment:
im not sure but do u need all of these checks? u cant just add query.trim().toLowerCase() in includes()?

Reply:
Hmmm yes, that`s a bit overkilled